db/schema.rb

```diff
@@ -525,65 +525,78 @@
     t.string "disability7"
     t.string "disability8"
     t.string "disability9"
+    t.string "itt_key"
+    t.string "rec_id"
+    t.string "status"
+    t.string "allocated_place"
+    t.string "provider_course_id"
+    t.string "initiatives_two"
+    t.string "ni_number"
+    t.string "numhus"
+    t.string "previous_surname"
+    t.string "surname16"
+    t.string "ttcid"
+    t.string "hesa_committed_at"
+    t.index ["hesa_id", "rec_id"], name: "index_hesa_students_on_hesa_id_and_rec_id", unique: true
```

Comment:
@HettieS 
This block and the `version` on line 13 should be the only changes in this file.

Reply:
@markhallen this should now be resolved